src/polytope/github/repository/Repository.py

```diff
@@ -9,14 +9,12 @@
 from polytope.github.RequestVerb import RequestVerb
 from polytope.github.Session import RequestsSession
 from polytope.github.Session import Session
+from polytope.github.Token import Token
```

Comment:
Now can we replace `"Token"` type from initializer of Requester and Repository with `Token`?

Reply:
Yes, we can. Thanks for your pointing out!
I will replace all `"Token"` with `Token`.